recipes-bsp/acpi-tables/acpi-tables.bb

```diff
@@ -14,6 +14,13 @@ inherit deploy
 
 ACPI_TABLES ?= ""
 ACPI_TABLES[doc] = "List of ACPI tables to include with the initrd"
+ACPI_FEATURES ?= "uart_2w spi i2c"
+IASLCC = " \
+    ${@bb.utils.contains('ACPI_FEATURES', 'uart_2w', '-D MUX_UART_2WIRE', '', d)} \
+    ${@bb.utils.contains('ACPI_FEATURES', 'uart_4w', '-D MUX_UART_4WIRE', '', d)} \
+    ${@bb.utils.contains('ACPI_FEATURES', 'i2c', '-D MUX_I2C', '', d)} \
+    ${@bb.utils.contains('ACPI_FEATURES', 'spi', '-D MUX_SPI', '', d)} \
```

Comment:
This should also include generic MUX_UART case, I suppose (for non-Edison boards).

Reply:
> This should also include generic MUX_UART case, I suppose (for non-Edison boards).

Not sure. Search for 'define' on all asl gives only /galileo/gpiomux-*.asl with:
    #define MUX_SPI
    #define MUX_I2C
    #define MUX_UART0

On galileo we are generating a table for each case. gpiomux.asl doesn't define anything which probably should stay that way by default.
So imho I should define MUX_UART0 + not initialize  ACPI_FEATURES ?= "uart_2w spi i2c", or only for edison.

Reply:
Ah, I see. Than it's fine. we might add an alias to Galileo case that would work as uart_2w and / or uart_4w.

Reply:
If the user has machine = galileo, and ACPI_TABLES ?= "gpiomux.asl", and ACPI_FEATURES ?= "MUX_SPI MUX_I2C MUX_UART0" he will get exactly the same as with ACPI_TABLES ?= "gpiomux-all.asl"

Reply:
I see. Thanks for explanation. Then  proceed as needed,